core/lib/spree/core/validators/email.rb

```diff
@@ -12,10 +12,8 @@ module Spree
   #     end
   #
   class EmailValidator < ActiveModel::EachValidator
-    EMAIL_REGEXP = /\A([^@\.]|[^@\.]([^@\s]*)[^@\.])@([^@\s]+\.)+[^@\s]+\z/
```

Comment:
It's ok to remove code from Solidus core, but since this is public and could be used in stores, we should deprecate its usage instead, and remove it in a future release. See this useful [gist](https://gist.github.com/rafaelfranca/5492531) and our own [Spree::Deprecation class](https://github.com/solidusio/solidus/blob/e878076f2ed670d07654ab6293a16588743f2fa6/core/lib/spree/deprecation.rb#L6).

Reply:
That makes sense, thanks for pointing it out.